docs/kb/semgrep-supply-chain/ssc-python-lockfiles.md

```diff
@@ -37,6 +36,8 @@ To generate a `requirements.txt` file from `requirements.in`, enter the followin
 pip-compile -o requirements.txt
 ```
 
+> Semgrep can also use the file if you change the extension of `requirements.txt` to `requirements.pip`.
```

Comment:
I don't think its important to highlight this. Much prefer to suggest generating a `*.txt` and nothing else.